Havit.Blazor.Components.Web.Bootstrap/TreeViews/HxTreeView.razor.cs

```diff
@@ -21,6 +21,16 @@ public partial class HxTreeView<TItem> : ComponentBase
 	/// Event fires when selected data item changes.
 	/// </summary>		
 	[Parameter] public EventCallback<TItem> SelectedItemChanged { get; set; }
+	/// <summary>
+	/// Event fires when expanding an item.
+	/// </summary>		
+	[Parameter] public EventCallback<TItem> ItemExpanded { get; set; }
+
+	/// <summary>
+	/// Event fires when collapsing an item.
+	/// </summary>		
+	[Parameter] public EventCallback<TItem> ItemCollapsed { get; set; }
+
```

Comment:
Parameters should be named `OnItemExpanded` and `OnItemCollapsed`.

(Only `XyChanged` callbacks do not use the `On` prefix in our conventions + some special cases such as `DataProvider`.)